Introduce performance tests and multi pipeline PoCs for metrics (#778)
kyma-project/community: concepts/observability-strategy/configurable-monitoring/pocs/tools/metric-gen/README.md

# metric-gen

The tool is used to generate dummy OTLP metrics and send them to a provided grpc OTLP endpoint endpoint (e.g. an OpenTelemetry collector with a configured `otlp` receiver).

## Usage

### Local

```bash
go run ./ -host=telemetry-otlp-metrics -port=4317
```

### Kubernetes

> One can also dockerize the tool and run it in a Kubernetes Deployment. This allows running multiple replicas in parallel to increase the produced load.

Reviewer: ```suggestion
You can also dockerize the tool and run it in a Kubernetes Deployment. This way, you can increase the produced load by running multiple replicas in parallel.
```
